bin/pd2pg

```diff
@@ -75,25 +75,28 @@ class PG2PD
   end
 
   # Adds users to user_schedule table associated with given schedule.
-  def user_schedules_to_db(items, schedule_id)
+  def user_schedules_to_db(schedule_items)
     columns = [:id, :user_id, :schedule_id]
-    records = items.map do |i|
-      ["#{i['id']}_#{schedule_id}",
-       i['id'],
-       schedule_id]
+    all_records = []
+    schedule_items.each do |s|
+      users = get_bulk(:users,
+                                "schedules/#{s['id']}/users",
+                                { since: Time.now.strftime('%Y-%m-%d') },
+                                false)
+      records = users.map do |u|
```

Comment:
Instead of that `map` and the subsequent `.concat`, if you need to add stuff to an existing list, prefer:

``` ruby
users.each do |u|
  all_records << ["#{u['id']}_#{s['id']}", u['id'], s['id']]
end
```

ooor (are you using `all_records` elsewhere?), just assign all_records to that map.

Reply:
Thanks for the rubonic suggestions!